source/Nrk.HttpRequester/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net452" />
-  <package id="Polly" version="5.0.5" targetFramework="net452" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net452" />
```

Comment:
Do we need a dependency on Newtonsoft.Json here?

Reply:
Don't think so, it's the later version of Microsoft.AspNet.WebApi.Client that's dependent on it, not the WebRequester directly.
Can remove. (Will do shortly)